### PR TITLE
The bleed update

### PR DESCRIPTION
**Pull request by QuietlyWhisper**
## About the PR

Uhhh okay so this PR changes a lot so bear with me. I originally started this because passive bleeding does basically nothing. In current gameplay, the max you can passively bleed is 1 unit per 5 seconds. This is because the bleeding was being reduced by an arbitrary hardcoded number. So I started by removing this number. Bleeding is now 1 to 1 ratio with the bleed rate. The next issue? **IT'S WAY TOO EASY TO MAX OUT BLEED RATE WITH THE CURRENT MODIFIERS.** A lizard can put you in max bleed by punching you twice. My solution? I reduced all bleed rate modifiers by 10x on all damage types. In my testing, this resulted in bleed being minimal in combat, but a larger impact in medical, as the faster you slow down the new 20u bleeding max, the less bloodloss damage you'll take over time. (It's actually really fun to handle bleeding now..!)

Because of all these changes, I additionally added a new item. The blood pack! This item is filled in most places you can find brute/bruise packs like vending machines and lockers, and will restore 10 blood per use, as well as doing very very minimal bleed rate reduction. They've also been added to cargo's medical supplies crate. Because of this new item, I've had to shuffle around some of the med kit fills as well. Changes will be in the changelog!

Iron has also been buffed to compensate for what _should_ be more blood loss, but the blood packs are more than enough to not **ALWAYS** require use of iron! I'll be looking to add a new saline chemical that is more effective than iron down the line.

The cautery now also functions, cauterizing your bleed rate by 5 units, at the cost of the patient taking 5 heat damage! Heat in general is more effective at cauterizing bleed, meaning lightbulbs, fires, laser guns... whatever can always be used as emergency means of cauterizing your wounds!

On the more coding side; I've taken the liberty of purging all mob YML files of their hardcoded default bleed values, so they inherit the default values (that were hardcoded) from the base mob! This means all downstream species and mobs (and future upstream mobs) can now more easily change bleed stats to whatever they feel is best!

More specifics in the changelog, but I figured I'd use this description space to outline the **VERY LARGE** scope of my PR.

Thanks for reading!

**Media**

**Most of the changes aren't visible, but here's the new item sprite, with thanks to TG!**
Screenshot 2023-03-23 164945

**And the full changelog, in no particular order, as I assume we don't want ALL of this in the changelogs.**
Screenshot 2023-03-23 174348

**Here's a more digestible view of the kit changes, although the blood pack sprite didn't exist at the time. I've marked the bruise pack and blood pack to better indicate two different items.**
image (1)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Whisper
- add: uhhh to be determined because I'm not sure how much of that changelog should be in here...

### CODE REVIEW

**Comment by QuietlyWhisper** [comment]
I'll need a smidge of help with the CL after the review...

File: Content.Server/Body/Systems/BloodstreamSystem.cs [hunk 1]
```diff
@@ -159,18 +161,26 @@ private void OnDamageChanged(EntityUid uid, BloodstreamComponent component, Dama
...
+        var prob = Math.Clamp(totalFloat / 25, 0, 1);
         var healPopupProb = Math.Clamp(Math.Abs(totalFloat) / 25, 0, 1);
         if (totalFloat > 0 && _robustRandom.Prob(prob))
         {
             TryModifyBloodLevel(uid, (-total) / 5, component);
             _audio.PlayPvs(component.InstantBloodSound, uid);
         }
+
+        // Heat damage will always cauterize, this message simply has a chance of showing up as feedback to avoid spam.
         else if (totalFloat < 0 && oldBleedAmount > 0 && _robustRandom.Prob(healPopupProb))
```
Comment (Veritius) [inline Content.Server/Body/Systems/BloodstreamSystem.cs:184]: This has a chance of not giving any feedback at all, or giving too much feedback.
Just add an accumulator to the component or something to stop popup spam.

[also this](https://cdn.discordapp.com/attachments/310555209753690112/1001323526223052881/ss13_code_shit.png)

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml [hunk 2]
```diff
@@ -67,6 +66,33 @@
...
+  - type: Tag
+    tags:
+    - Bloodpack
+  - type: Sprite
+    state: Bloodpack
+  - type: Healing
+    damageContainer: Biological
+    damage:
+      types:
+        bloodloss: -0.5 #for whatever godforsaken reason I cannot touch blood level without healing one of these, so fuck it, bleed rate healing
```
Comment (Veritius) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:84]: what

Comment (QuietlyWhisper) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:84]: Anger

File: Content.Server/Body/Systems/BloodstreamSystem.cs (hunk 1 above)
Comment (QuietlyWhisper) [reply Content.Server/Body/Systems/BloodstreamSystem.cs:184]: It's just the feedback fortunately... while less of an issue it's still absolutely an issue, and not one I know how to fix...

**Comment by QuietlyWhisper** [comment]
Not sure what these current test fails are...

**Comment by QuietlyWhisper** [comment]
This was a bitch to fix for no reason at all but I think it should be fine now

**Comment by Emisse** [comment]
fuck it we test merge

**Pull request merged**